src/main/groovy/org/labkey/gradle/plugin/UiTest.groovy

```diff
@@ -77,25 +77,26 @@ class UiTest implements Plugin<Project>
     protected void addDependencies(Project project)
     {
         if (project.path != ":server:test")
-            BuildUtils.addLabKeyDependency(project: project, config: 'uiTestCompile', depProjectPath: ":server:test", depVersion: project.labkeyVersion)
+            BuildUtils.addLabKeyDependency(project: project, config: 'uiTestImplementation', depProjectPath: ":server:test", depVersion: project.labkeyVersion)
 
-        String schemasProjectPath = BuildUtils.getProjectPath(project.gradle, "schemasProjectPath", ":schemas")
+        String schemasProjectPath = BuildUtils.getSchemasProjectPath(project.gradle)
         if (project.findProject(schemasProjectPath) != null)
-            BuildUtils.addLabKeyDependency(project: project, config: 'uiTestCompile', depProjectPath: schemasProjectPath, depVersion: project.labkeyVersion)
-        BuildUtils.addLabKeyDependency(project: project, config: 'uiTestCompile', depProjectPath: BuildUtils.getProjectPath(project.gradle, "apiProjectPath", ":server:api"), depVersion: project.labkeyVersion)
-        BuildUtils.addLabKeyDependency(project: project, config: 'uiTestCompile', depProjectPath: BuildUtils.getProjectPath(project.gradle, "remoteApiProjectPath", ":remoteapi:java"), depVersion: project.labkeyVersion)
+            BuildUtils.addLabKeyDependency(project: project, config: 'uiTestImplementation', depProjectPath: schemasProjectPath, depVersion: project.labkeyVersion)
+        BuildUtils.addLabKeyDependency(project: project, config: 'uiTestImplementation', depProjectPath: BuildUtils.getApiProjectPath(project.gradle), depVersion: project.labkeyVersion)
+        BuildUtils.addLabKeyDependency(project: project, config: 'uiTestImplementation', depProjectPath: BuildUtils.getRemoteApiProjectPath(project.gradle), depVersion: project.labkeyVersion)
```

Comment:
Could we remove the schemas and API dependencies from the plugin and define them in SVN instead? We're working on removing test dependencies on product code and it would be nice to get the plugin update portion of the story out of the way now.

Reply:
I shall attempt this.  It may mean that running the tests for an individual module doesn't work without also including these dependencies as well, but it's not something that's really supported or used now anyway.